src/components/NewForm/QuestionForm.js

```diff
@@ -0,0 +1,133 @@
+import { Button, Card, Form, Input, Select } from "antd";
+import { makeTag } from "../../utils/utility";
+import Choices from "./Choices";
+import { useEffect, useState } from "react";
+
+const QuestionForm = (props) => {
+  const { questions, form, onAddQuestion } = props;
+  const [newQuestion, setNewQuestion] = useState(false);
+  const [showOptions, setShowOptions] = useState(false);
+  const { Option } = Select;
+
+  function handleCurrentQuestion(event) {
+    form.setFieldValue("question", event.target.value);
+  }
+
+  useEffect(() => {
+    if (form.getFieldValue("inputType") === undefined) {
+      form.setFieldValue("inputType", "string");
+    }
+  });
+
+  function addQuestion() {
+    setNewQuestion(true);
+  }
+
+  function handleSaveQuestion() {
+    let inputType = form.getFieldValue("inputType");
+    console.log("All values", form.getFieldsValue());
+    let choices = form.getFieldValue("choices");
+    let newQuestion = {
+      question: form.getFieldValue("question"),
+      answerType: inputType,
+      tag: makeTag(6),
```

Comment:
static value to makeTag?

Reply:
if it represents something, use enum

Reply:
will use enum, but will skip for this PR